### DISCUSSION TITLE
allow filtering pull requests by if the author is a bot (`author_is_bot:<bool>`)

### DISCUSSION THREAD

**Discussion by thesuperzapper** [community/community#122852] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I think we should be filter pull requests (and issues) by if they were created by a bot.

This is a big deal for large projects that use bots like [`dependabot`](https://github.com/dependabot), as they can create hundreds of PRs and make it hard for feature / bug-fix PRs to get attention from maintainers.

Once some kind of `author_is_bot:<bool>` filter is added, I think this filter should be highlighted in the default "open pr" view, perhaps as a checkbox:

<img width="1304" alt="Screenshot 2024-05-07 at 17 53 33" src="https://github.com/community/community/assets/5735406/c60dceb4-d2c4-4acd-a34c-da49044980ef">

**Comment by mjfpalmer** [comment]
If you only want to exclude PRs from a single bot, this can be done by adding `-author:app/dependabot` to the filter for example, _but_ there doesn't seem to be a way to exclude multiple bot authors.

If I try `-author:app/dependabot -author:app/someotherbot`, the second is stripped off. I've tried with and without ANDs/ORs etc. and different approaches without luck. Has anyone found a way to do this?

So, I'm definitely for a filter that excludes all bot PRs!

UPDATE: the `-author:app/dependabot -author:app/someotherbot` syntax works https://github.com/search screen, but not in https://github.com/pulls
